chips/nrf52/src/ficr.rs

```diff
@@ -130,6 +130,8 @@ register_bitfields! [u32,
         PART OFFSET(0) NUMBITS(32) [
             /// nRF52832
             N52832 = 0x52832,
+            /// nRF52833
+            N52833 = 0x52833,
```

Comment:
could make sense to port the [nRF52833-DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52833-DK) too then in the future I guess